state_machine/applications/system-check/tests/logging_test.py

```diff
@@ -0,0 +1,17 @@
+"""
+Python system check script for PyCubed satellite board
+PyCubed Mini mainboard-v02 for Pocketqube Mission
+SD Card Logging Test
+* Author(s): Yashika Batra
+"""
+
+# copy and paste logging functions and use them
+
+def run(cubesat, hardware_dict, result_dict):
+    # if no SD Card detected, update result dictionary and return
+    if not hardware_dict['SDcard']:
+        result_dict['SD_Card_Logging'] = ('Cannot test logging; no SD Card detected', False)
+        return result_dict
+
```

Comment:
Is this incomplete at the moment?

Reply:
Yep, it's not complete at the moment because i was/am still ediitng the SD card functions. I'm mainly done with those though (they're on the misc repo), so I'll probably define a simple test before the next update. 

It'll likely just create a file, make sure it exists and has a timestamp, and then delete the file and make sure it's no longer there.